templates/zerver/help/view-zulip-version.md

```diff
@@ -0,0 +1,46 @@
+# View Zulip version
+
+## Zulip server and web app version
+
+If your organization self-hosts Zulip, the server administrator will need to
+[upgrade Zulip][upgrade-zulip] in order for you to experience the latest
+improvements. You can check which [version][changelog] of the Zulip server and
+web app your organization is using.
+
+Zulip Cloud organizations are always updated to the latest version of Zulip.
+
+[upgrade-zulip]:
+    https://zulip.readthedocs.io/en/latest/production/upgrade-or-modify.html
+[changelog]: https://zulip.readthedocs.io/en/latest/overview/changelog.html
+
+### View Zulip server and web app version
+
+{start_tabs}
+
+1. Click on the **gear** (<i class="fa fa-cog"></i>) icon in the upper
+   right corner of the web or desktop app.
+
+1. Select **About Zulip** from the menu.
+
+1. View the version number under **Zulip Server**.
+
+{end_tabs}
+
+## Zulip desktop app version
+
+The [Zulip desktop app](/apps/) has an independent version from the server and
+webapp.
+
+### View Zulip desktop app version
+
+{start_tabs}
+
+1. Click on the **Help** menu in the desktop app. The app version is shown in the
+   dropdown: **Zulip Desktop v...**
+
+{end_tabs}
+
+## Related articles
+
+* [Desktop installation guides](/help/desktop-app-install-guide)
+* [Upgrading Zulip][upgrade-zulip]
```

Comment:
We should link https://zulip.readthedocs.io/en/latest/overview/release-lifecycle.html somewhere from this page.